main.ts

```diff
@@ -44,7 +58,7 @@ export default class NLSyntaxHighlightPlugin extends Plugin {
 		const colors = [settings.adjectiveColor, settings.nounColor, settings.adverbColor, settings.verbColor, settings.conjunctionColor];
 
 		for (let i = 0; i < partsOfSpeech.length; i++) {
-			if (enabled[i]) {
```

Comment:
Rather than simply disabling the styling, can we unload the CodeMirror extension entirely?

You can do this by removing the extension from `NLSyntaxHighlightPlugin.extensions` and calling `app.workspace.updateOptions()`.

See https://github.com/artisticat1/nl-syntax-highlighting/blob/4bee4d127d14e9a9395d867f39424d43ac6a630e/main.ts#L92-L97 for reference.